ehr/test/src/org/labkey/test/tests/onprc_ehr/AbstractGenericONPRC_EHRTest.java

```diff
@@ -97,6 +97,13 @@ protected EHRClientAPIHelper getApiHelper()
         return new EHRClientAPIHelper(this, getContainerPath());
     }
 
+
+    @Override
+    protected boolean skipStudyImportQueryValidation()
+    {
+        return true;
```

Comment:
Is this still needed?

Reply:
Is this for test length / performance reasons?  Is at least one of the ONPRC EHR tests doing a query validation?